app/src/main/java/com/github/polypoly/app/menu/ProfileModifyingActivity.kt

```diff
@@ -37,7 +39,7 @@ class ProfileModifyingActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         val id = intent.getLongExtra("userId", 0)
-        val user = FakeRemoteStorage.instance.getUserWithId(id).get()
+        val user = FakeRemoteStorage.instance.getValue<User>(DB_USERS_PROFILES_PATH + id).get()
```

Comment:
Calling `.get()` on every future kind of beats the whole purpose of having futures...

Reply:
This applies everywhere of course